src/components/Toast/Toast.tsx

```diff
@@ -106,23 +118,35 @@ const Toast: FC<ToastProps> & { types?: typeof ToastType; actionTypes?: typeof T
   const iconElement = !hideIcon && getIcon(type, icon);
 
   return (
-    <CSSTransition in={open} classNames="monday-style-toast-animation" timeout={400} unmountOnExit>
-      <div className={classNames} role="alert" aria-live="polite">
-        {iconElement && <div className="monday-style-toast-icon">{iconElement}</div>}
+    <CSSTransition
+      in={open}
+      classNames={{ enterActive: styles.enterActive, exitActive: styles.exitActive }}
```

Comment:
this one is a breaking change because we removing "monday-style-toast-animation" no? and why (just wonder) we split to different classes for enter and exit?

Reply:
I believe it's not really dangerous to remove it (I doubt someone override it) but still thought it worth mention

Reply:
Not sure how to make it overridable in this case...